packages/upgrade/.gitignore

```diff
@@ -1,2 +1,3 @@
 # Generated by `esbuild` using `yarn build`
 /cli.js
+/fixtures/sample-project/example-imports-to-unified-package.js
```

Comment:
What's going on with this file btw? I wasn't sure if it should be tracked or not

Reply:
The goal was that the file would be added, but then we wouldn't track any further changes to the file. So you could run codemods on it but never accidentally push an updated file containing the output of the codemod.

This doesn't work though. We could use git's `--skip-worktree` instead, but if the file is changed, `git pull` may fail and `git status` will show nothing has changed. Might be best to just comment in the file "don't commit changes to this file"?